### ISSUE TITLE
[Bug]: can't add a custom network on iPhone

### ISSUE THREAD

**Issue by kl3eo** [MetaMask/metamask-mobile#8344]
### Describe the bug

The button "Add" is deep down and cannot be reached with swipe screen (the top warning against malicious network prevents it to be seen on default)

### Expected behavior

Expect to be able to "Add" a custom network

### Screenshots/Recordings

idiotic behaviuor

### Steps to reproduce

1. Go to Add custom network

### Error messages or log output

```shell
mobile iphone
```

### Version

latest

### Build type

None

### Device

iphone

### Operating system

iOS

### Additional context

tried but zero

### Severity

fucking critical

**Comment by DanielTech21** [comment]
Hi @kl3eo 

Thanks for reaching out to MetaMask Technical Support.

Please take a look at this guide on how to add a custom network.
https://support.metamask.io/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC

Thanks.

**Comment by plasmacorral** [comment]
@kl3eo Could you please attach a screenshot?

**Comment by knl** [comment]
Not the original author, but this is what I get. Note that scrolling down to reveal "Add" button is not sticky, screen will auto scroll to the top. As iPhone is single touch, I can't scroll down and click on "Add".

PS. That tiny blue thing at the bottom is the "Add" button.

![Screenshot 2024-01-24 at 23 31 21](https://github.com/MetaMask/metamask-mobile/assets/361496/70a7dd7e-9d18-4023-9f4a-132d23045f5b)

**Comment by kl3eo** [comment]
I removed/re-installed MM, then tried again. The same stuff, although i was eventually looking for chance and happened to finger the Block Explorer URL field and here we go the "Add" button becomes visible and clickable. But this is "optional" and without knowing it you'd stick in "no button to click" position.

**Comment by ewqz** [comment]
A smaller font size fixed it.
Go to "Settings > Display & Brightness > Text Size".
![IMG_1025](https://github.com/MetaMask/metamask-mobile/assets/150484190/69219e37-8120-4b9d-824d-cccfc386e2af)

**Comment by plasmacorral** [comment]
Thanks for the screenshot @knl that is helpful, and thanks for the additional context @ewqz.  This reduces accessibility and we'll get it over to the right team to help get it improved.
